.travis.yml

```diff
@@ -1,23 +1,36 @@
-os:
-- linux
-sudo: false
-language: python
-python:
-- "3.6"
-install: pip install tox-travis
-script: tox
+jobs:
+  include:
+    - stage: flake8
```

Comment:
Cannot find selenium stage, did we add it into the CI process?

Reply:
We will add this when selenium tests are written.